### PR TITLE
[O2-2082] Do not include FairLogger.h

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Review by davidrohr — CHANGES_REQUESTED** [review]
Inside O2, we have Framework/Logger.h which should be used instead of fairlogger/Logger.h directly.

**Comment by Barthelemy** [comment]
ah right, forgot it, let me change it

File: Detectors/FIT/FDD/base/CMakeLists.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 o2_add_library(FDDBase
                SOURCES src/Geometry.cxx
-               PUBLIC_LINK_LIBRARIES O2::CommonConstants ROOT::Geom FairRoot::Base)
+               PUBLIC_LINK_LIBRARIES O2::CommonConstants ROOT::Geom FairRoot::Base O2::Framework)
```
Comment (sawenzel) [inline Detectors/FIT/FDD/base/CMakeLists.txt:13]: why is this needed? This will bring a whole tail of new library dependencies.

Comment (sawenzel) [reply Detectors/FIT/FDD/base/CMakeLists.txt:13]: If it's just for the logger. We should probably make a dedicated lib.

**Comment by davidrohr** [comment]
There is already a separate library for that `FrameworkLogger`
@Barthelemy : Could you switch to that?

**Comment by Barthelemy** [comment]
Given that the QC CI is now green, it seems that this PR is not necessary any more (for me). I will close it but if you feel that I should proceed with the change (and using `FrameworkLogger`) let me know.
